src/actions/dot/migrate/index.ts

```diff
@@ -146,7 +159,7 @@ const migrate = async (opts: Options) => {
         continue;
       }
 
-      if (txType === "claimAttest") {
+      if (txType === "claimAttest" || (module === 'claims' && txType === "claim")) {
```

Comment:
This is because the indices pallet has a `claim` function too, right?

Reply:
yes, we need to caught that too, but there is a problem when injecting